core/descriptors/src/org/jetbrains/kotlin/descriptors/ClassDescriptor.java

```diff
@@ -79,6 +79,10 @@
     @ReadOnly
     List<ReceiverParameterDescriptor> getAdditionalReceivers();
 
+    @NotNull
+    @ReadOnly
+    List<PropertyDescriptor> getPropertiesForAdditionalReceivers();
```

Comment:
Do you think it's hard or not to avoid adding this method?
It's only used within a couple of places in IR while being exposed to all usages of `ClassDescriptor`